tailwind.config.mjs

```diff
@@ -7,7 +7,20 @@ const defaultTheme = require("tailwindcss/defaultTheme");
 /** @type {TailwindConfig} */
 export default {
   mode: "jit",
-  content: ["./app/**/*.{ts,tsx}", "./data/**/*.md", "./tailwind-extras.html"],
+  content: ["./app/**/*.{ts,tsx}", "./data/**/*.md"],
```

Comment:
Thanks for this solution!

I don't actually know if we need this at all, since we're already looking at `.md` in our `content`

I think we can remove the `safelist` as well and go ahead and merge this in